force-app/main/default/lwc/marketingCloudCopadoDataTable/marketingCloudCopadoDataTable.js

```diff
@@ -308,9 +308,11 @@ export default class MarketingCloudCopadoDataTable extends LightningElement {
                     "/events/copado/v1/step-monitor/"
                 )
             ) {
-                // show progress on screen
-                const stepStatus = JSON.parse(response.data.payload.copado__Payload__c);
-                this.progressStatus = stepStatus.data.progressStatus || this.progressStatus;
+                if (this.isJSON(response.data.payload.copado__Payload__c)) {
+                    // show progress on screen
+                    const stepStatus = JSON.parse(response.data.payload.copado__Payload__c);
```

Comment:
what is actually wrong with copado__Payload__c's value? did output it via console.log?